TheFourFifths/consus-client
# [TFF-170] Update client functional tests to work as integration tests

### Summary

This PR updates all client functional tests to be runnable as integration tests.

Sister PRs (merge **this** PR second!):
* TheFourFifths/consus/pull/69 (merge first!)
* TheFourFifths/consus-integration/pull/2 (merge last!)

### Checklist


### How to Review

1. Check out the `functional-integration-TFF-170` branch on `consus`
2. `npm install -g` in `consus`
3. `consusd start --port=8081` anywhere
4. Check out the `functional-integration-TFF-170` branch on `consus-client`
5. `npm install` in `consus-client`
6. `npm run integration-test` in `consus-client`
7. `consusd stop` anywhere
8. Verify that all tests pass

### Links

- [TFF-29](https://msoese.atlassian.net/browse/TFF-29)
- TheFourFifths/consus-integration/pull/2
- TheFourFifths/consus/pull/69
